pkg/download/bucket/testdata/buckets.json

```diff
@@ -33,7 +36,18 @@
             "status": "active",
             "retentionDays": 462,
             "metricInterval": "PT1M",
-            "version": 1
+            "version": 1,
+            "updatable": false
+        },
+        {
+            "bucketName": "my-bucket-name-that-shall-not-be-downloaded",
+            "table": "metrics",
+            "displayName": "Default metrics (15 months)",
+            "status": "deleting",
+            "retentionDays": 462,
+            "metricInterval": "PT1M",
+            "version": 1,
+            "updatable": true
```

Comment:
as you mentioned that older versions might not return that field, please don't forget to reintroduce some test configs that do not include it!

Reply:
Added